bin/init_repo.sh

```diff
@@ -37,12 +40,21 @@ for template_file in "${template_files[@]}" ; do
         for idx in "${!user_values[@]}" ; do
             placeholder=${placeholders[idx]}
             placeholder_name="${placeholder%%:*}"
-            result=`echo "$result" | sed "s/{{$placeholder_name}}/${user_values[idx]}/g"`
+            result=`echo "$result" | sed "s#{{$placeholder_name}}#${user_values[idx]}#g"`
+            if [ "$placeholder_name" = "SRC" ] && [ ${user_values[idx]} != 'src' ]; then
+              mv "$current_path/../src" "$current_path/../${user_values[idx]}"
+            fi
         done
 
-        echo "$result" > "$dest_path"
-        printf "${green}Processed and created ${nc}$dest_path\n"
-        rm "$source_path"
+        if [ -n "${result}" ];
+        then
+          echo "$result" > "$dest_path"
+          arrPath=(${dest_path//\// })
```

Comment:
This bit seems quite convoluted.
Shouldn't be better to extend the model [here](https://github.com/CGnal/python-ci-template/blob/main/bin/init_repo.sh#L9) with a colon separator and include the dest path? Something like `source_file:dest_path_relative_to_root`.

Reply:
I like this idea, I'll do it.